app/controllers/api/base_controller.rb

```diff
@@ -1,78 +1,5 @@
 class Api::BaseController < ApplicationController
-  class SchemaValidationError < StandardError ; end
```

Comment:
This code was basically moved to `SchemaValidatable` (a controller concern) and `JsonSchemaValidator` (a PORO).